creator-node/.gitignore

```diff
@@ -15,3 +15,4 @@ coverage/
 .nyc_output/
 compose/env/commonEnv.sh
 file_storage/
+tmp*/
```

Comment:
I think this should be `compose/env/tmp*/` ?

Reply:
Good catch!